.github/checklist.yml

```diff
@@ -0,0 +1,10 @@
+paths:
+  "Cargo.lock":
+    - Did you review changed dependencies for security concerns?
+    - Has at least one other team member approved the dependency changes?
+  "rfcs/**":
+    - Have at least 3 team members approved this RFC?
+  "src/**":
+    - Did you add adequate behavior tests (`tests/behavior`)? Specifically, nested fields support.
+    - Did you add adequate logging? Note: we support rate-limited logging for the hot path.
```

Comment:
We could probably phrase this in a way that would cause people to think about it a little more. E.g. "For each failure path, is there sufficient context logged for users to investigate the issue?" or something like that.